tools/markdown-magic/src/transforms/includeCodeTransform.cjs

```diff
@@ -0,0 +1,69 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const fs = require("fs");
+const {
+	formattedEmbeddedContentBody,
+	formattedSectionText,
+	readFile,
+	resolveRelativePath,
+} = require("../utilities.cjs");
+
+/**
+ * Embeds contents from the specified file paths within the provided (optional) line boundaries.
+ *
+ * @param {object} content - The original document file contents.
+ * @param {object} options - Transform options.
+ * @param {string} options.path - Relative path from the document to the file being embedded.
+ * @param {string | undefined} options.start - (optional) First line from the target file to be embedded (inclusive).
+ * Expected to be a string-formatted integer.
+ * Default: 0.
+ * Constraints are the same as those for the `end` parameter to
+ * {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice#parameters | Array.slice}
+ * @param {string | undefined} options.end - (optional) Line of the target file at which to end the embedded range (exclusive).
+ * Expected to be a string-formatted integer.
+ * Default: <file-line-count> + 1.
```

Comment:
I think this would be the correct default under the current logic (0-based indexing) but I actually like it being 1-indexed, thus `+ 1` here, but also start defaulting to 1.
```suggestion
 * Default: <file-line-count>.
```

Reply:
I may actually rephrase this as "includes through the end of the file", rather than expressing in terms of indices.